config/resources/mandaat-domain.lisp

```diff
@@ -67,6 +71,9 @@
              (organization          :via ,(s-prefix "org:hasMember")
                                     :inverse t
                                     :as "organization")
+             (user                  :via ,(s-prefix "sign:IsOndertekenaarVoor")
```

Comment:
`sign:IsOndertekenaarVoor`: voor predicates gebruiken we conventioneel lowercase als eerste letter, voor classes uppercase. Pas je dat aan?